[
  {
    "text": "Hi, the same GraphQL query perfectly runs on the GraphQL tool but is not working on the API “<https://api.hubapi.com/collector/graphql>”. What could be the issue? Please help me on this\n```query MyQuery {\n  CRM {\n    contact(\n      uniqueIdentifier: \"email\"\n      uniqueIdentifierValue: \"***@****.**\"\n    ) {\n      email\n      associations {\n        ticket_collection__wp_client_portal_display_contact_ticket(\n          offset: 0\n          orderBy: createdate__desc\n          limit: 10\n          filter: {}\n        ) {\n          total\n          limit\n          offset\n          hasMore\n          items {\n            HubSpot_owner_id\n            subject\n            source_type\n            closed_date\n            content\n            createdate\n            hs_pipeline_stage\n            hs_pipeline\n            hs_ticket_priority\n            hs_object_id\n          }\n        }\n      }\n    }\n  }\n}```\n“message”: “CRM_SEARCH_RECORD_FETCH_LIMIT_ERROR: Retrieving more than 10,000 records is not supported by CRM Search API due to system limitations.“,\n      “locations”: [\n        {\n          “line”: 6,\n          “column”: 11,\n          “sourceName”: null\n        }\n      ],\n      “path”: [\n        “/CRM/contact_collection/items[0]/associations/ticket_collection__wp_1_client_portal_display_contact_ticket”\n      ],",
    "time_stamp": "2025-02-12 06:41:55"
  },
  {
    "poster": {
      "poster_name": "Manab Roy",
      "poster_handle": "manab986"
    },
    "time_stamp": "2025-02-12 07:41:12",
    "text": "Is there any problem with the GraphQL tool? I created a few association labels 30 minutes ago, but the GraphQL schema was not updated. I am not able to run any query.",
    "reply_count": 1,
    "reply_users_count": 1,
    "replies": [
      {
        "time_stamp": "2025-02-12 07:43:40"
      }
    ]
  },
  {
    "text": ":yawning_face:",
    "time_stamp": "2025-02-12 07:43:40"
  }
]